tensorboard/data/experimental/experiment_from_dev.py

```diff
@@ -80,6 +86,9 @@ def get_scalars(self, runs_filter=None, tags_filter=None, pivot=None):
                 "tags_filter support for get_scalars() is not implemented yet."
             )
         pivot = True if pivot is None else pivot
+        include_wall_time = (
+            False if include_wall_time is None else include_wall_time
+        )
```

Comment:
Is there a reason why you are setting default values to `None` rather than `True`/`False`?

Using a default value of `None` is recommended if the default would otherwise be a mutable variables (e.g., an empty `list` or `dict`), but booleans are immutable. It is clearer to use `True`/`False` directly than to set them inside the method.

Sometimes there are legitimate use-cases for defaulting a boolean variable to `None`, for detecting if a parameter was explicitly passed or not. But I don't think they apply here. (And if they do apply in the future, you can always change the default value later.)

Reply:
Insisting that _all_ default values are `None` really is not standard practice in Python (e.g., you won't find it in Google's Python Style Guide). It doesn't really matter, but you might try inquiring with g/python-users or g/python-style for opinions of other experienced Python programmers.

Here's my take:

1. This extensibility concern about keeping default values in sync is only relevant if you're using _implementation inheritance_, but implementation inheritance is a maintenance nightmare that should be avoided in almost all case. If you're not using implementation inheritance, then you'll have to duplicate the logic for default values anyways.
2. Open source and data science users in particular are much more likely to see public _interfaces_ for code (e.g., with auto-complete or `SomeClass.some_method?` in a notebook) than Google engineers who will go straight to the original source code to figure out what it does. Even if there is a maintenance cost associated with duplicating default values, it would be worth it to make interfaces more self-documenting.

Reply:
Definitely agree about avoiding implementation inheritance, but the
extensibility concern is also relevant with composition; any kind of
wrapper function that just needs to forward arguments will run into
this. (I run into this fairly frequently, and having to write tests that
ensure that the `inspect.signature`s are kept in sync is pretty
annoying.) I don’t have super strong opinions about this and won’t block
over it, but I do think that the `None`-default form stands on its
merits—and I consider myself an open source user first and foremost. :-)

Reply:
Thank you both for the discussion, William and Stephan. I'll take the liberty to be the tie breaker here :) 
I'll err on the side of making life easier for open-source users and change the default values to explicit `True`/`False`. An additional argument is that it's confusing for generated docs to say something like "pivot=None". This argument is similar to the notebook/IDE auto-complete argument, but is also somewhat standalone because there are users who don't use IDE but still look at docs.

Reply:
Done. PTAL.